docker-compose.yml

```diff
@@ -5,6 +5,13 @@ services:
     image: okteto/bin:1.1.2
     build:
       context: .
+      target: opensource
+      dockerfile: bin/Dockerfile
+  bin-cloud:
```

Comment:
are we going to be able to build this with okteto build too?

Reply:
Yes
```
% okteto build --help
Build (and optionally push) a Docker image

Usage:
  okteto build [flags]

Flags:
  -f, --file string     name of the Dockerfile (Default is 'PATH/Dockerfile')
  -h, --help            help for build
      --no-cache        do not use cache when building the image
  -t, --tag string      name and optionally a tag in the 'name:tag' format (it is automatically pushed)
      --target string   set the target build stage to build
```